### ISSUE TITLE
discussion: misleading contributions

### ISSUE THREAD

**Issue by BenjaSanchez** [computer-aided-biotech/better-cb#7]
As discussed with @dcsoto, there's a small issue with having all of the [rootstock](https://github.com/manubot/rootstock) history in this repo, as the "Contributors" panel of the repo looks [misleading](https://github.com/computer-aided-biotech/better-cb/graphs/contributors) (most of those people have not contributed to this manuscript). We could squash all old commits up to [`ce827c7`](https://github.com/computer-aided-biotech/better-cb/commit/ce827c7789163dca33dfa3bd2df7aa828e57485d) in a single commit, however I realized this has a big drawback: doing so would conflict this repo with rootstock, so we would not be able to pull changes from them in the future  to include their latest features.

@dcsoto should we then squash? How about instead offering a file that self records the contributions by number of words written, in a similar style than figure 2 of [the manubot publication](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007128)? There are tools out there like [Perceval](https://github.com/chaoss/grimoirelab-perceval) that can do this automatically already, so we could even add it to the CI cycle (at the deployment stage) as a file recorded in the `output` branch. I could also open an issue over at https://github.com/manubot/rootstock to ask if this is a feature they have considered adding.

**Comment by dcsoto** [comment]
Good point. Are we really concern with getting their latest features tho? I feel the tool works fine as it is for our current needs. However, I do like the idea of Perceval, using it to define the order of contributors sounds good to me :)

Yay to opening an issue in the manubot/rootstock repo

Would you like to work setting up Perceval?

**Comment by BenjaSanchez** [comment]
update: offline it was decided with @dcsoto to leave the repo how it is for now, and only once we start sharing with more people the final version we will squash the past :)
